tensilelite/Tensile/Source/lib/include/Tensile/PropertyMatching.hpp

```diff
@@ -975,6 +975,19 @@ namespace Tensile
                 bool      Debug = T_Debug;
                 std::cout << std::setprecision(2) << std::fixed;
 
+                Key key = key_orig;
+                if(!Debug::Instance().gridBasedBatchExp())
```

Comment:
Looks like this is enabled by default ( "!" in if condition)
If you would like to enable this by default, probably it is better not to use "Debug".

Reply:
@yenong-amd ask me to leave env variable to use previous behavior, in case there is regression or you want to do more test on gridbased tuning with batch.
But if we decide to abandon gridbased tuning with batch, we can remove it.

Reply:
I mean, this name is not best describing what this does.
You use the same name but changed the meaning (opposite behavior as default).

Reply:
That means if you want to do gridbased tuning with batch experiment, you should use this env.

Reply:
OK. I was talking about the naming, not about the usage.
It is not a big deal. We can keep this name.